jenkins/validate-artifacts/validate-artifacts.jenkinsfile

```diff
@@ -157,10 +172,16 @@ pipeline {
                         for (distribution in distributionList) {
                             def local_architecture = architecture.trim()
                             def local_distribution = distribution.trim()
+                            def local_platform = local_distribution == "zip" ? "windows" : "linux"
```

Comment:
We should not do this but rather follow up with the key-value pair.

Reply:
Sure Peter I'll make the required changes.

Reply:
Hey @peterzhuamazon , Added requested changes along with version bump.The PR is ready for review now.
Thanks.